src/docfx/build/Build.cs

```diff
@@ -41,24 +41,29 @@ public static int Run(string workingDirectory, CommandLineOptions options)
 
         private static bool BuildDocset(string docsetPath, string? outputPath, CommandLineOptions options, FetchOptions fetchOptions)
         {
-            using var errorLog = new ErrorLog(outputPath);
             var stopwatch = Stopwatch.StartNew();
 
+            using var errorLog = new ErrorLog(outputPath);
+            using var disposables = new DisposableCollector();
+
             try
             {
                 var configLoader = new ConfigLoader(errorLog);
-                var (errors, config, buildOptions, packageResolver, fileResolver) = configLoader.Load(docsetPath, outputPath, options, fetchOptions);
-                if (errorLog.Write(errors))
```

Comment:
When building a _loc repo_ with _multiple docsets_ when _some docsets_ are not provisioned, this code base return here, causing `packageResolver` not disposing immediately.

In this case `packageResolver` holds a reader lock of fallback repo, and that blocks all other docsets.